components/region_cache_memory_engine/src/background.rs

```diff
@@ -242,6 +300,35 @@ impl Runnable for BackgroundRunner {
                 }
                 self.gc_finished();
             }
+            BackgroundTask::LoadTask => {
```

Comment:
Should we spawn this as an async task in a background worker remote like we do with full compaction (which also involves an expensive rocksdb operation): https://github.com/tikv/tikv/blob/66847e9c5ac701e73da1366938a7e2306cacc671/components/raftstore/src/store/worker/compact.rs#L371

I believe, the answer here is no, as it would not be safe to perform a gc concurrently with a load, and (unlike full compaction) we do not intend to pause this thread. However, just wanted to check. Thoughts @tonyxuqqi ?

Reply:
Only the load of the range completes will the range be GCed. Here is a background worker, we may not need it to be done in another worker.

Reply:
yes, but there is only one thread here. since GC and load cannot run concurrently. see how I did this for full compaction. this can be done later (i can do it), just leave a TODO.

Reply:
Load itself will be quite expensive and it may block GC, which is critical for in-memory-engine's performance. So I think it's better to have a background task for the load.

Reply:
All right, it has been doen.